framework/i18n/Formatter.php

```diff
@@ -578,16 +577,30 @@ public function asImage($value, $options = [])
      * Formats the value as a hyperlink.
      * @param mixed $value the value to be formatted.
      * @param array $options the tag options in terms of name-value pairs. See [[Html::a()]].
+     * @param bool|string $scheme the URI scheme to use in the formatted hyperlink (available since 2.0.43):
+     *
+     * - `false (default)`: adding non-secure protocol scheme if there is none added already
+     * - `true`: adding secure protocol scheme if there is none added already
+     * - string: adding the specified scheme (either `http`, `https` or empty string
+     *   for protocol-relative URL) if there is none added already
+     *
      * @return string the formatted result.
      */
-    public function asUrl($value, $options = [])
+    public function asUrl($value, $options = [], $scheme = false)
```

Comment:
Looks to me like a BC-break.
Shouldn't the default be `null` and leave the $url unchanged?

Reply:
The previous behavior was to add `http://` 😏 so it works like that also now.

Reply:
Totally misread the diff, apologies 🤡